.dockerignore

```diff
@@ -1,13 +1,20 @@
-**/node_modules/
-**/node_modules_linux/
+**/.circleci/
+**/.vscode/
+**/.idea/
+**/coverage/
 **/docs/
-**/Dockerfile
+**/node_modules/
+**/node_modules_linux/g
 **/.dockerignore
 **/.editorconfig
-**/.eslintrc.js
-**/.gitignore
 **/.git
-**/README.md
+**/.gitattributes
+**/.gitignore
+**/.travis.yml
+**/CODE-OF-CONDUCT.md
+**/Dockerfile
 **/LICENSE
-**/.vscode
-**/.idea
+**/README.md
+**/docker-compose.yml
```

Comment:
Same thing here, include this, right?

Reply:
Same answer. The image doesn't need the blueprint anymore when it gets built.

Reply:
Based on your comment above, this is probably fine too.